resources/views/livewire/missing-media-search.blade.php

```diff
@@ -1,10 +1,42 @@
 <section class="panelV2">
-    <h1 class="panel__heading">Missing Media</h1>
+    <header class="panel__header">
+        <h2 class="panel__heading">Missing Media</h2>
+        <div class="panel__actions">
+            <div class="panel__action">
+                <div class="form__group">
+                    <input
+                        id="name"
+                        class="form__text"
+                        type="text"
+                        wire:model.live="name"
+                        placeholder=" "
+                    />
+                    <label class="form__label form__label--floating" for="name">
+                        {{ __('torrent.title') }}
+                    </label>
+                </div>
+            </div>
+            <div class="panel__action">
+                <div class="form__group">
+                    <input
+                        id="year"
+                        class="form__text"
+                        type="number[4]"
```

Comment:
Typo here. Also, try and use text types instead of numeric to avoid the value incrementing/decrementing when scrolled over.
```
type="text"
class="form__text"
inputmode="numeric"
pattern="[0-9]*"
```

Reply:
Updated.